### PR TITLE
Do Debian installation verification with hello-world image.

### PR DESCRIPTION
**Pull request by thewtex**
The hello-world image is recommended verification image and it is smaller
than Ubuntu.

### CODE REVIEW

**Comment by thewtex** [comment]
Signed -- thanks for your review.

**Comment by moxiegirl** [comment]
LGTM

ping @docker/docs-owners

**Comment by thaJeztah** [comment]
Perhaps this should even be changed to `docker run --rm hello-world`; the `hello-world` image is created for this task (showing that the installation is working) and is even smaller

**Comment by thewtex** [comment]
@thaJeztah good idea.  Push a new revision that uses hello-world.

File: docs/sources/installation/debian.md [hunk 1]
```diff
@@ -28,9 +28,10 @@ To install the latest Debian package (may not be the latest Docker release):
 
 To verify that everything has worked as expected:
 
-    $ sudo docker run -i -t ubuntu /bin/bash
+    $ sudo docker run --rm hello-world
 
-Which should download the `ubuntu` image, and then start `bash` in a container.
+which should download the `hello-world` image, and then print a friendly Hello
```
Comment (moxiegirl) [inline docs/sources/installation/debian.md]: This command downloads and runs the `hello-world` image in a container. When the container runs, it prints an informational message about your Docker installation. Then, it exits.

Comment (thewtex) [inline docs/sources/installation/debian.md]: Thanks for the suggestion.

The hello-world container does not contain any information about the local Docker installation, so I will tweak that wording.

**Comment by moxiegirl** [comment]
LGTM

ping @docker/docs-owners

**Comment by thaJeztah** [comment]
Thanks for updating the PR; I don't think we need to explain the `--rm` in this context, so LGTM

**Pull request merged**